srcpkgs/rust/template

```diff
@@ -58,10 +58,21 @@ else
 		 https://static.rust-lang.org/dist/rust-std-${_rust_dist_version}-i686-unknown-linux-gnu.tar.gz
 		 https://static.rust-lang.org/dist/cargo-${_cargo_dist_version}-i686-unknown-linux-gnu.tar.xz"
 		checksum+="
-		 549116e23dce8687f69327f5ea2c717c93175d24d82d456fab37160f800f8ce8
-		 14de38321f6f2c64496e77bac793ad5511d37696b9c33283bb6f395dc590c4d9
+		 45e633c985c259a13d3b096662241bd8255a86ed4917e0fec396e7071c04c734
+		 69bd6064f605c1d0d7d41070efea1e329d75755b04fbc5a13ffc53efdc3a97c7
 		 200cf76636ccbb153d5c430a5c0f3ad5af206899f76470e3cd046679a5a4ab3f"
 		;;
+	ppc64le)
+		distfiles+="
+		 https://static.rust-lang.org/dist/rustc-${_rust_dist_version}-powerpc64le-unknown-linux-gnu.tar.xz
+		 https://static.rust-lang.org/dist/rustc-${_rust_dist_version}-powerpc64le-unknown-linux-gnu.tar.xz
+		 https://void-ppc64.octaforge.org/distfiles/cargo-${_cargo_dist_version}-powerpc64le-unknown-linux-gnu.tar.xz"
```

Comment:
Should this be mirrored on Void infra @Gottox?

Reply:
I believe all should be mirrored. I mean, I can host my distfiles forever, but stuff like this should probably be mirrored.

Reply:
So....@Gottox?

Reply:
@Cogitri files are on /distfiles, can you change the URLs?